lib/db/mysql/index.js

```diff
@@ -153,7 +153,7 @@ MysqlStore.prototype = {
   addAvatar: function addAvatar(id, uid, url, provider, selected) {
     id = buf(id);
     uid = buf(uid);
-    var store = this;
+    var store = this; // eslint-disable-line consistent-this
```

Comment:
ESLint i think usually prefers `var self = this;` for consistency: http://eslint.org/docs/rules/consistent-this

But we can disable it inline, or we can disable the rule globally in the .eslintrc file if you prefer.